### PR TITLE
Ci grab bag 18 apr 24

### PR DESCRIPTION
**Pull request by aimurphy**
Thanks for contributing to the Harness Developer Hub! Our code owners will review your submission.

## Description

* Please describe your changes:Some small requests that came up when I was on PTO. Docker Hub as default Harness Image store. Clarify that 2000 free build credits are only for free plans.
* Jira/GitHub Issue numbers (if any): CI-12008, CI-12007
* Preview links/images (Internal contributors only): *building*

## PR lifecycle

We aim to merge PRs within one week or less, but delays happen sometimes.

If your PR is open longer than two weeks without any human activity, please tag a [code owner](https://github.com/harness/developer-hub/blob/main/.github/CODEOWNERS) in a comment.

PRs must meet these requirements to be merged:

- [ ] Successful preview build.
- [ ] Code owner review.
- [ ] No merge conflicts.
- [ ] Release notes/new features docs: Feature/version released to at least one prod environment.

### CODE REVIEW

File: docs/continuous-integration/get-started/ci-subscription-mgmt.md [hunk 1]
```diff
@@ -103,7 +103,9 @@ Based on the values in the table above:
 
 ### Credit allowance by plan tier
 
-All plans receive 2000 free credits every month. These credits can be consumed by all users within the account registered to run builds on Harness Cloud. Any free credits that are unused at the end of the month expire automatically and do not roll over from one month to the next.
+Free plans receive 2000 free credits every month. Credits for Team and Enterprise plans are factored into the plan terms. These credits can be consumed by all users within the account registered to run builds on Harness Cloud.
+
+For free plans, any free credits that are unused at the end of the month expire automatically and do not roll over from one month to the next.
```
Comment (brian-f-harness) [inline docs/continuous-integration/get-started/ci-subscription-mgmt.md:108]:
```suggestion
For free plans, any unused credits at the end of the month expire automatically and do not roll over from one month to the next.
```

File: docs/platform/connectors/artifact-repositories/connect-to-harness-container-image-registry-using-docker-connector.md [hunk 2]
```diff
@@ -25,6 +26,35 @@ To prevent rate limiting or throttling issues when pulling images, configure the
...
+   If there is no connector with the `harnessImage` identifier in your Account, you need to [create a Docker connector](/docs/platform/connectors/cloud-providers/ref-cloud-providers/docker-registry-connector-settings-reference) with the exact **Id** of `harnessImage`. Harness gives precedence to the connector with the `harnessImage` identifier and uses it to pull the images.
+
+3. Select **Edit Details**.
+4. Select **Continue** to go to the **Details** settings.
+5. For **Provider Type** and **URL**, do one of the following:
+
+   * To pull [Harness images from GCR](https://console.cloud.google.com/gcr/images/gcr-prod/global/harness), select **Other (Docker V2 compliant)** for **Provider Type**, and then enter `https://gcr.io/gcr-prod` for **Docker Registry URL**.
+   * To pull [Harness images from ECR](https://gallery.ecr.aws/harness), select **Other (Docker V2 compliant)** for **Provider Type**, and then enter `https://gallery.ecr.aws` for **Docker Registry URL**.
+
+   If you want to change the connector back to Docker Hub, select **Docker Hub** and entering `https://registry.hub.docker.com`.
```
Comment (brian-f-harness) [inline docs/platform/connectors/artifact-repositories/connect-to-harness-container-image-registry-using-docker-connector.md:47]:
```suggestion
   If you want to change the connector back to Docker Hub, select **Docker Hub** and enter `https://registry.hub.docker.com`.
```

**Review by brian-f-harness — APPROVED** [review]
LGTM

File: docs/continuous-integration/get-started/ci-subscription-mgmt.md (hunk 1 above)
Comment (aimurphy) [reply docs/continuous-integration/get-started/ci-subscription-mgmt.md:108]: Want to separate the free ones because they might buy additional credits.

**Pull request merged**